server/src/routers/index.ts

```diff
@@ -10,5 +11,6 @@ rootRouter.use('/auth', authRouter);
 rootRouter.use('/categories', categoryRouter);
 rootRouter.use('/items', itemRouter);
 rootRouter.use('/wishs', wishRouter);
+rootRouter.use('/cart', cartRouter);
```

Comment:
```suggestion
rootRouter.use('/carts', cartRouter);
```
다른 것과 통일성있게 얘도 복수형으로 가면 좋을 거 같아요~ 😄

Reply:
넵 좋습니다!